forgerock-openbanking-uk-aspsp-rs/forgerock-openbanking-uk-aspsp-rs-mock-portal/forgerock-openbanking-uk-aspsp-rs-mock-portal-server/src/main/java/com/forgerock/openbanking/rs/ui/api/data/DataEventsApi.java

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2019 ForgeRock AS.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.forgerock.openbanking.rs.ui.api.data;
+
+import com.forgerock.openbanking.common.model.data.FRDataEvent;
+import com.forgerock.openbanking.common.model.openbanking.forgerock.event.FREventNotification;
+import com.forgerock.openbanking.exceptions.OBErrorException;
+import com.forgerock.openbanking.exceptions.OBErrorResponseException;
+import io.swagger.annotations.*;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.Collection;
+
+@Api(tags = "DataEvents", description = "the data events export/import API")
+@RequestMapping(value = "/api/data")
+public interface DataEventsApi {
+
+    @ApiOperation(
+            value = "Export all data events",
+            notes = "Export the all events data as OB format",
+            response = FREventNotification.class,
+            responseContainer = "List",
+            authorizations = {
+                    @Authorization(value = "SSOToken")
+            },
+            tags = {"DataEvents",}
+    )
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Export all events data",
+                    response = FREventNotification.class),
+    })
+    @RequestMapping(
+            value = "/events/all",
```

Comment:
You don't need to worry about versions in the URL?

Reply:
We don't need a versioning for a customer feature at least not yet.